k8s.gcr.io/manifests/k8s-staging-cluster-api-do/promoter-manifest.yaml

```diff
@@ -0,0 +1,11 @@
+# google group for gcr.io/k8s-staging-cluster-api-do is k8s-infra-staging-cluster-api-do@kubernetes.io
+registries:
+- name: gcr.io/k8s-staging-cluster-api-do
+  src: true
+- name: us.gcr.io/k8s-artifacts-prod/cluster-api-do
```

Comment:
@detiber Have you given thought to "nesting" of cluster-api in GCR-space?

Reply:
@thockin It seems like there is some discussion ongoing here https://github.com/kubernetes/k8s.io/pull/658#issuecomment-601421263. However, I believe that is out of the scope of this PR and out of the scope for CAPI-DO maintainers.